Core/Frameworks/Flake/Controller/HtmlBlockTemplated.php

```diff
@@ -42,4 +52,9 @@ function render() {
 
         return $sHtml;
     }
+
+    function execute()
+    {
+        // TODO: Implement execute() method.
```

Comment:
An `execute()` method is needed because `\Flake\Core\Controller` has an abstract `execute()`
I guess we will survive with it having no code, since the system works without it at the moment.